zips/zip-0271.md

```diff
@@ -141,7 +152,10 @@ to
 ## Extension to the lockbox funding stream
 
 The ``FS_DEFERRED`` lockbox funding stream is set to receive
-12% of the block subsidy and is extended until block height 4406400.
+12% of the block subsidy and is extended by 1,260,000 blocks on
+both Mainnet and Testnet. If this ZIP activates before or on the first block after the
+end of current dev fund as of this writing and specified in [^zip-1015-funding-streams],
+this would extend ``FS_DEFERRED`` on Mainnet until block height 4406400.
```

Comment:
ZIP 1016 requires that the new funding streams start at the end of the current dev funds (placing a constraint on the latest that NU6.1 can activate). I don't think we can say that if it activates late, it still goes for 3 chain-years.